### DISCUSSION TITLE
Cronjob Link just for Mailmagik

### DISCUSSION THREAD

**Discussion by chocwithstarfish** [creecros/Mailmagik#39] [Q&A] [answered]
Hello, I'm currently using the predefined URL for the Kanboard cron job (https://domain.tld/?controller=CronjobController&action=run&token==WEBHOOK_TOKEN_HERE) which executes all scheduled tasks, including email notifications. I am interested in whether there is a way to specifically target the Mailmagik email fetch function with a cron job link, so I can run it every 10 minutes independently of the full cron job, which I prefer running once daily. Any guidance on setting up a more frequent, specific cron job for Mailmagik would be greatly appreciated.

**Comment by creecros** [comment]
Yes, use the supplied fetchmail 

See the readme.

**Comment by chocwithstarfish** [comment]
Thank you for the suggestion to use ./cli mailmagik:fetchmail. However, I'm looking to trigger this command through a URL for a web-based cron service, which requires a specific URL to hit. Is there a way to set up or convert the Mailmagik fetchmail command to a URL endpoint so that I can use it with my web-based cron service for more frequent, specific task fetching?

**Comment by creecros** [comment]
Yes, but it would require a controller.

Here is the existing controller, you would need to change the command.

https://github.com/kanboard/kanboard/blob/5649bf017e46c5852b767667c49deb91c2bfe42b/app/Controller/CronjobController.php#L23

**Comment by chocwithstarfish** [comment]
Unfortunately, I'm not technically proficient enough to make modifications to the controller as suggested. I lack the skills for creating, implementing, or calling modified files, and I fear I might cause more harm than good. This leads me to conclude that perhaps this software might not be the optimal solution for my needs. Thank you for the responses and all the best.

**Reply by alfredbuehler** [reply]
This might be a reasonable new feature for Mailmagik. Let me think about it.

**Comment by alfredbuehler** [comment, marked as answer]
The cron URL for Mailmagik is available since version [1.6.1](https://github.com/creecros/Mailmagik/releases/tag/1.6.1).
